templates/poster.html

```diff
@@ -24,18 +24,21 @@
 <!-- Title -->
 <div class="pp-card m-3" style="">
   <div class="card-header">
-    <h2 class="card-title main-title text-center" style="">
+    <h2 class="card-title main-title text-center" style="color: black">
       {{paper.content.title}}
     </h2>
     <h3 class="card-subtitle mb-2 text-muted text-center">
       {% for author in paper.content.authors %}
-      <a href="papers.html?filter=authors&search={{author}}" class="text-muted"
+      <a href="papers.html?filter=authors&search={{author}}" class="text-primary"
         >{{author}}</a
       >{{ "," if not loop.last }}
       {% endfor %}
     </h3>
     <p class="card-text text-center">
-      <span class="">Keywords:</span>
+      {% set key_title = "Keywords:"%}
+      {% if paper.content.keywords != blank %}
```

Comment:
this seems do not work? I still see the `Keywords: ` in the outcome

Reply:
yes :/  I will remove it